frontend/components/icons/ExternalLink.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const ExternalLink = () => {
+  return (
+    <svg width="12" height="13" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I just had a conversation today about how we are going to export SVGs from figma. We want to export the container the SVG is in which will ensure we always have `16px x 16px` dimensions for our icons. I think here we've exported small icon itself.

Sorry for the delay in letting you know but can we change so the 16x16 container is used here instead. same for the chevron icon

Reply:
This is what it looks like when we have a 16x16 container:

<img width="606" alt="Screen Shot 2022-10-21 at 3 01 09 PM" src="https://user-images.githubusercontent.com/71795832/197270330-f130a8e8-72b9-4005-a668-5a458a60b394.png">

If we are ensuring all icons are 16x16, even though our text is 12pt. We need to have a size="small" to make sure its resized because this extra padding around the 12x13 is causing alignment issues when it's a text icon.

Reply:
The 16x16 is only for the new SVG icons going forward. We probably have some different dimensions currently and should chat with Mike T about what they dimensions for a smaller icon will be. In guessing 8x8